machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -3220,10 +3628,37 @@ def jetsystematics(self):
                     error_var_down = 0 # absolute downward uncertainty for a given category in a given (pt_jet, shape) bin
                     count_sys_up = 0
                     count_sys_down = 0
+                    error = 0
                     for sys_var in range(self.systematic_variations[sys_cat]):
                         # FIXME exception for the untagged bin pylint: disable=fixme
                         bin_first = 2 if "untagged" in self.systematic_varlabels[sys_cat][sys_var] else 1
-                        error = input_histograms_sys[ibin2][sys_cat][sys_var].GetBinContent(ibinshape + bin_first) - input_histograms_default[ibin2].GetBinContent(ibinshape + 1)
+                        if self.use_inclusive_systematics:
+                            if self.systematic_catnames[sys_cat] == "cuts":
+                                #file_inc = TFile.Open(self.inclusive_unc)
+                                #unc_hist = file_inc.Get("rms_mult_1")
+                                #unc = unc_hist.GetBinContent(unc_hist.FindBin(self.lpt_finbinmin[0]))
+                                unc = 0.05
+                                default = input_histograms_default[ibin2].GetBinContent(ibinshape+1)
+                                error = unc*default
+                            elif self.systematic_catnames[sys_cat] == "cutvar":
+                                #file_inc = TFile.Open(self.inclusive_unc)
+                                #unc_hist = file_inc.Get("rms_mult_1")
+                                #unc = unc_hist.GetBinContent(unc_hist.FindBin(self.lpt_finbinmin[0]))
+                                unc = 0.1
+                                default = input_histograms_default[ibin2].GetBinContent(ibinshape+1)
+                                error = unc*default
+                            else:
+                                # FIXME exception for the untagged bin pylint: disable=fixme
+                                if input_histograms_sys[ibin2][sys_cat][sys_var].Integral()==0:
+                                    error = 0
```

Comment:
these places where the variation failed should be set to negative so the error dont count in the RMS denominator

Reply:
Fixed